node/src/manager/commands/stats.rs

```diff
@@ -165,3 +174,28 @@ pub fn target(
     }
     Ok(())
 }
+
+pub fn set_target(
+    store: Arc<SubgraphStore>,
+    primary: ConnectionPool,
+    search: &DeploymentSearch,
+    entity: Option<&str>,
+    columns: Vec<String>,
+    target: i32,
+    no_analyze: bool,
+) -> Result<(), anyhow::Error> {
+    let columns = if columns.is_empty() {
+        vec!["id".to_string(), "block_range".to_string()]
+    } else {
+        columns
+    };
```

Comment:
Two questions:
1. Should `id` and `block_range` always be included?
2. What about `vid`?

Reply:
I haven't seen `vid` be useful here, that's why I left it out. And I felt that not adding to a list of user-provided columns is more understandable than always adding them.